app/controllers/application_controller.rb

```diff
@@ -1,4 +1,8 @@
 class ApplicationController < ActionController::API
+  include ActionController::HttpAuthentication::Token::ControllerMethods
+
+  respond_to :json
```

Comment:
Why including the module? It seems it can be used directly: http://api.rubyonrails.org/classes/ActionController/HttpAuthentication/Token.html